Assets/SEE/Game/City/EdgeMeshScheduler.cs

```diff
@@ -186,7 +186,7 @@ public void OnNext(ChangeEvent value)
             if (value is EdgeEvent { Change: ChangeType.Addition } edgeEvent)
             {
                 // If this is an added edge, we are going to need to turn it into a mesh.
-                LoadingSpinner.Show(LoadingText);
+                //LoadingSpinner.Show(LoadingText);
```

Comment:
I think a loading spinner in general still makes sense here, since edges being generated are a user-visible process. IIRC, the code here is probably commented out because the loading spinner doesn't disappear. Maybe we should create a follow-up issue for this.

Reply:
I reverted this change. The compiler example can be run with this LoadingSpinner on.